lib/active_ldap/railtie.rb

```diff
@@ -7,7 +7,7 @@
 module ActiveLdap
   class Railtie < Rails::Railtie
     initializer "active_ldap.deprecator", before: :load_environment_config do |app|
-      app.deprecators[:active_ldap] = ActiveLdap.deprecator
+      app.deprecators[:active_ldap] = ActiveLdap.deprecator if defined?(app.deprecators)
```

Comment:
Could you use `respond_to?`?

```suggestion
      app.deprecators[:active_ldap] = ActiveLdap.deprecator if app.respond_to?(:deprecators)
```